iroh-net/src/relay/http.rs

```diff
@@ -12,6 +12,18 @@ pub(crate) const HTTP_UPGRADE_PROTOCOL: &str = "iroh derp http";
 pub(crate) const WEBSOCKET_UPGRADE_PROTOCOL: &str = "websocket";
 pub(crate) const SUPPORTED_WEBSOCKET_VERSION: &str = "13";
 
+/// The HTTP path under which the relay accepts relaying connections
+/// (over websockets and a custom upgrade protocol).
+pub const RELAY_PATH: &str = "/relay";
+/// The HTTP path under which the relay allows doing latency queries for testing.
+pub const RELAY_PROBE_PATH: &str = "/relay/probe";
```

Comment:
So this is just bikesheeding, but nesting `/relay/probe` is a bit odd, given the other handlers are all on the root (I appreciate this was copied from tailscale).  I'd move this to `/probe` so it is on the same level as `/generate_204` etc.  We can definitely do this for the new route.  We can even remove `/derp/probe` if we want as it's not yet used (hi https-probe that still lives in a half-dead branch :sweat: ).

Reply:
I'll keep it in this PR as a refactor instead of a change - let's talk about moving it when we're making use of it :)